modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -172,6 +174,24 @@ namespace cv {
                     fused_layer_names.push_back(last_layer);
                 }
 
+                void setDropout()
+                {
+                    cv::dnn::LayerParams dropout_param;
+                    dropout_param.name = "Dropout-name";
+                    dropout_param.type = "Dropout";
+                    darknet::LayerParameter lp;
+
+                    std::string layer_name = cv::format("dropout_%d", layer_id);
+                    lp.layer_name = layer_name;
+                    lp.layer_type = dropout_param.type;
+                    lp.layerParams = dropout_param;
+                    lp.bottom_indexes.push_back(last_layer);
+                    last_layer = layer_name;
+                    net->layers.push_back(lp);
+                    layer_id++;
+                    fused_layer_names.push_back(last_layer);
+                }
```

Comment:
@YashasSamaga, thanks for PR! Do you mind to let @ashishkrshrivastava to integrate Dropout layer with https://github.com/opencv/opencv/pull/16438? So the rest of layers will be yours :)

Reply:
That'll be great. I am anyway having trouble finding free time to complete the PR. I have been very busy for the last two weeks. It'll be until next Wednesday before I can make progress here.